src/module/token/minting/MintableERC1155.sol

```diff
@@ -112,22 +104,12 @@ contract MintableERC1155 is Module, EIP712, BeforeMintCallbackERC1155, OnTokenUR
     /// @dev Emitted when the minting request signature is unauthorized.
     error MintableRequestUnauthorized();
 
-    /*//////////////////////////////////////////////////////////////
-                                EVENTS
-    //////////////////////////////////////////////////////////////*/
-
-    /// @dev Emitted when a token's metadata URI is updated.
-    event MintableTokenURIUpdated(uint256 tokenId, string tokenURI);
-
-    /// @notice Emitted when the metadata URI for a token is updated.
-    event MetadataUpdate(uint256 id);
-
     /*//////////////////////////////////////////////////////////////
                                 CONSTANTS
     //////////////////////////////////////////////////////////////*/
 
     bytes32 private constant TYPEHASH_SIGNATURE_MINT_ERC1155 = keccak256(
-        "MintRequestERC1155(uint256 tokenId,uint48 startTimestamp,uint48 endTimestamp,address recipient,uint256 quantity,address currency,uint256 pricePerUnit,string metadataURI,bytes32 uid)"
+        "MintRequestERC1155(uint256 tokenId,uint48 startTimestamp,uint48 endTimestamp,address recipient,uint256 quantity,address currency,uint256 pricePerUnit,bytes32 uid)"
```

Comment:
you planning to update 1155 in this PR too? or separate?

Reply:
would prob keep it in separate PRs tbh. will be easier to review

Reply:
Let's leave the changes in so far and I'll cut a new branch off of this one since it needs some of the new files introduced